catalyst-gateway/bin/src/cli.rs

```diff
@@ -48,8 +52,48 @@ impl Cli {
             Self::Run(settings) => {
                 logger::init(settings.log_level)?;
 
+                // Tick every N seconds until config exists in db
+                let check_config_tick = env::var("CHECK_CONFIG_TICK")?;
```

Comment:
suggestion: make this variables as a part of `ServiceSettings` (maybe make sense to separate them into the different struct as it was done for the `DocsSettings`). So it would be explicitly visible from the CLI how to configure these parameters.

Reply:
Good idea, I changed this.